starters/solidstart-tanstackquery-tailwind-modules/src/components/CounterExample/Counter.spec.tsx

```diff
@@ -19,14 +19,16 @@ describe('CounterExample', () => {
     await render(() => <CounterExample />);
     const button = await screen.getByText('Decrement');
     fireEvent.click(button);
-    await expect(screen.getByText('Count: -1')).toBeDefined()
+    await expect(screen.getByText('Count: 0')).toBeDefined();
+    fireEvent.click(button);
+    await expect(screen.getByText('Count: -1')).toBeDefined();
   });
 
   it('should mount and reset', async () => {
     await render(() => <CounterExample />);
     const DecrButton = await screen.getByText('Decrement');
     fireEvent.click(DecrButton);
-    await expect(screen.getByText('Count: -1')).toBeDefined()
+    await expect(screen.getByText('Count: -2')).toBeDefined()
```

Comment:
There is no need for the decrement btn again. just the rest test is okay since we have test for decrement already

Reply:
And not sure why await is in front of expect. 
Please remove. It should still work